test/features/HelloWorld.feature

```diff
@@ -0,0 +1,6 @@
+@HelloWorld-Feature
```

Comment:
why do you need to add this tag?

Reply:
I saw an example, they placed them in every feature they were going to test, but that is when they want to divide different scenarios in one feature right?

Reply:
yep, I think that for now, it is not needed